addons/test_mass_mailing/tests/test_mailing_test.py

```diff
@@ -158,3 +165,39 @@ def test_mailing_test_equals_reality(self):
             'test@test.com',
             "Should use the value of the previous record's email_to as default",
         )
+
+    def test_mailing_test_correct_model(self):
```

Comment:
:socks: Order tests alphabetically in a given class (as it is the execution order, better to follow it)